pages/oauth/authorize.js

```diff
@@ -59,7 +59,11 @@ const OAuthAuthorizePage = () => {
   const queryParams = { skip: skipQuery, variables: queryVariables, context: API_V2_CONTEXT };
   const { data, error, loading: isLoadingAuthorization } = useQuery(applicationQuery, queryParams);
   const isLoading = loadingLoggedInUser || isLoadingAuthorization;
-  const requestedScopes = query.scope ? query.scope.split(',').map(s => s.trim()) : [];
+  const requestedScopes = query.scope
+    ? decodeURIComponent(query.scope)
+        .split(/,|\s/)
```

Comment:
Arguably we may want to do `\s+`, or we may want to do the map / trim logic in the API as well.